src/Config_Command.php

```diff
@@ -1035,9 +1035,9 @@ private static function fetch_remote_salts( $insecure = false ) {
 	 *
 	 * @param string $name
 	 * @param string $type
-	 * @param string $type
+	 * @param array $values
```

Comment:
Can we make the type more precise? I think this one is correct
```suggestion
	 * @param array<array{name: string, value: string, type: string}> $values
```